bin/i18n/sync-all.rb

```diff
@@ -138,15 +138,16 @@ def should_i(question)
     end
   end
 
-  def return_to_staging_branch
+  def return_to_staging_branch(force: false)
     case GitUtils.current_branch
     when "staging"
       # If we're already on staging, we don't need to bother
       return
     when /^i18n-sync/
-      # If we're on an i18n sync branch, only return to staging if the branch
-      # has been merged.
-      return unless GitUtils.current_branch_merged_into? "origin/staging"
+      unless force
```

Comment:
maybe I'm missing something, but it seems like this functionality will be the same? If we're just adding a force flag and a condition on it.

Reply:
`return unless GitUtils.current_branch_merged_into?('origin/staging')` can prevent switching to the staging branch.
And the main idea is to skip all the logic that can prevent us from doing that except in situations like:
https://github.com/code-dot-org/code-dot-org/blob/staging/bin/i18n/sync-all.rb#L151-L153